### ISSUE TITLE
Amazing work!

### ISSUE THREAD

**Issue by jiayukk** [ysharma3501/NovaSR#5]
Thank you for your amazing work! 
I would like to ask whether your training only involved downsampling from 48k to 16k, or if there were other data augmentation measures as well?
What kind of schemes did you use for the discriminator? Could you recommend some references?

**Comment by ysharma3501** [comment]
@jiayukk Thanks for the compliment!

1. It was mostly that and I randomly added some quantization noise as well for v1.

2. For training, infact no discriminators are used which massively speeds up training. Strictly using spectral losses and phase losses was enough since the model architecture is surprisingly robust at least for 16khz to 48khz. 

Training code will probably come today or tomorrow as well for reference.

**Comment by ysharma3501** [comment]
Training code: https://www.kaggle.com/code/yatharthsharma888/novasr-training

**Comment by jiayukk** [comment]
Thanks for sharing code. 
I’m interested in the design of ”MultiResPhaseTrioLoss“. 
Did you come up with it yourself or was it inspired by existing work? 
Could you kindly provide me with some references?

**Comment by jiayukk** [comment]
The overall workflow of the model is:
Input: 16 kHz waveform
Step 1: Use F.interpolate (linear interpolation) to upsample from 16 kHz → 48 kHz
Step 2: The network further refines the upsampled waveform

Compared to directly using F.interpolate to go from 16 kHz to 48 kHz without the later refinement step, how much improvement does the additional processing bring?
Would you compare improvements using metrics such as LSD?

**Comment by ysharma3501** [comment]
@jiayukk The phase loss was inspired from Apnet and yes I am testing out benchmarks against other models and linear resample itself. 

You can try playing samples with only linear resample too, they will sound muffled so the neural network definitely has a considerable improvement.
